src/raft.rs

```diff
@@ -534,41 +531,186 @@ impl<T: Storage> Raft<T> {
         is_batched
     }
 
+    // Whether the leader can send raft logs through Follower Replication
+    // and returns the group id of the given peer.
+    #[inline]
+    fn use_delegate(&self, to: u64) -> Option<u64> {
+        if self.is_follower_replication_enabled()
+            && self.is_leader()
+            && self
+                .groups
+                .get_members(to)
+                .map_or(0, |members| members.len())
+                > 1
+            && !self.groups.in_same_group(self.id, to)
+        {
+            self.groups.get_group_id(to)
+        } else {
+            None
+        }
+    }
+
+    #[inline]
+    fn is_follower_replication_enabled(&self) -> bool {
```

Comment:
I think it's not accurate. Every peer can't know `follower replication` is enabled or not. It just choose a strategy after receives followers' messages.

Reply:
We can add some comments to make this clear.